cypress/integration/common/common.js

```diff
@@ -0,0 +1,27 @@
+import { Given, Then } from 'cypress-cucumber-preprocessor/steps'
+import { dashboards } from '../../assets/backends'
+
+beforeEach(() => {
+    cy.visit('/', {
+        timeout: 15000,
+    })
+})
+
+Given('I open the {string} dashboard', title => {
+    cy.get('[data-test="dhis2-uicore-chip"]', {
+        timeout: 15000,
+    })
+        .contains(title)
```

Comment:
I would use an explicit version of `contains` to avoid double matches for e.g. `My title` and `My title extended` which would cause the test to fail (since `click` needs a single target)

Reply:
Sorry I didnt realise this was the dashboard title, there can only be one of course.